mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -1385,6 +1386,88 @@ vectorizeAsLinalgGeneric(RewriterBase &rewriter, VectorizationState &state,
 
   return success();
 }
+// Vectorize an `tensor::UnPackOp` without OuterDimsPerms to these 4 Ops:
+//   Vector::TransferReadOp - Reads the Vector Array of Source data
+//   vector::TransposeOp - Transpose the Source
+//   ShapeCastOp - Reshapes the data based on the target.
+//   vector::TransferWriteOp. - Write the result vector back.
```

Comment:
nit: `//` -> `///` for doc. Also remove empty line between the doc and the function so that Doxygen picks it.

Reply:
Fixed.